From 63153f460bb60b7b2330427a3f32c3ab3b16aadb Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vrabaud@willowgarage.com>
Date: Sat, 2 Mar 2013 19:31:09 -0800
Subject: [PATCH 6/7] change the includedir for backward compatibility

---
 orocos-bfl.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/orocos-bfl.pc.in b/orocos-bfl.pc.in
index 6e8cd26..c1efd60 100644
--- a/orocos-bfl.pc.in
+++ b/orocos-bfl.pc.in
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 libdir=${prefix}/lib
-includedir=${prefix}/include
+includedir=${prefix}/include/bfl
 
 Name: orocos-bfl
 Description: Orocos Bayesian Filtering Library
-- 
1.7.12.4 (Apple Git-37)

