From 23fadf0722d22771d538f80499028e02c4a2e0d5 Mon Sep 17 00:00:00 2001
From: William Woodall <william@osrfoundation.org>
Date: Mon, 1 Jul 2013 18:09:31 -0700
Subject: [PATCH 7/7] Fix a compile issue on OS X

---
 src/filter/mixtureParticleFilter.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/filter/mixtureParticleFilter.cpp b/src/filter/mixtureParticleFilter.cpp
index 83170a4..10a30a4 100644
--- a/src/filter/mixtureParticleFilter.cpp
+++ b/src/filter/mixtureParticleFilter.cpp
@@ -102,15 +102,15 @@ MixtureParticleFilter<SV,MV>::MixtureParticleFilter(Mixture<SV> * prior,
   */
   bool ret = (dynamic_cast<MCPdf<SV> *>(this->_post))->ListOfSamplesSet(prior->ListOfSamplesGet());
   assert(ret);
-  for(int i =0 ; i < prior.NumComponentsGet() ; i++)
+  for(int i =0 ; i < prior->NumComponentsGet() ; i++)
   {
     bool ret = (dynamic_cast<MCPdf<SV> *>(this->_post->ComponentGet(i)))->ListOfSamplesSet(prior->ComponentGet(i)->ListOfSamplesGet());
   }
 
   // Initialise vector of lists of samples
-  for(int i =0 ; i < prior.NumComponentsGet() ; i++)
+  for(int i =0 ; i < prior->NumComponentsGet() ; i++)
   {
-    _old_samplesVec[i] = (prior.ComponentGet(i)->ListOfSamplesGet());
+    _old_samplesVec[i] = (prior->ComponentGet(i)->ListOfSamplesGet());
   }
   _new_samplesVec = _old_samplesVec;
 
-- 
1.7.12.4 (Apple Git-37)

